plugins/modules/user_role_workflow_manager.py

```diff
@@ -1056,14 +1056,13 @@ def valid_user_config_parameters(self, user_config):
         """
         self.log("Validating user configuration parameters...", "INFO")
         error_messages = []
+        regex_name_validation = re.compile(r"^[A-Za-z0-9_-]+$")
 
-        first_name_regex = re.compile(r"^[A-Za-z0-9_-]+$")
-        self.validate_string_field(user_config.get("first_name"), first_name_regex, "first_name: 'first_name' must only contain letters, numbers, \
-                                   underscores, and hyphens and should not contain spaces or other special characters.", error_messages)
+        self.validate_string_field(user_config.get("first_name"), regex_name_validation, "first_name: 'first_name' must only contain letters, numbers, "
```

Comment:
self.validate_string_field(user_config.get("first_name"), regex_name_validation, 
            "first_name: 'first_name' {0}".format(self.regex_name_validation), error_messages)

        self.validate_string_field(user_config.get("last_name"), regex_name_validation, 
            "last_name: 'last_name' {0}".format(self.regex_name_validation), error_messages)

Reply:
Updated the code.